question-5/question-5.php

```diff
@@ -7,11 +7,32 @@
  * 1. Register custom post type
  */
 
-// Your code here
+// Register Custom Post Type
+function lion_interview() {
+  $args = array(
+    'public'                => false,
+    'show_ui'               => true,
+    'show_in_menu'          => true,
+  );
+  register_post_type( 'post', $args );
+
+}
+add_action( 'init', 'lion_interview', 0 );
 
 
 /**
  * 2. Register taxonomy
  */
 
-// Your code here
\ No newline at end of file
+// Register Custom Taxonomy
+function lion_position() {
+  $args = array(
+    'public'                     => false,
+    'show_ui'                    => true,
+    'show_admin_column'          => true,
+    'show_in_nav_menus'          => true,,
+  );
+  register_taxonomy( 'taxonomy', array( 'lion_interview' ), $args );
```

Comment:
```suggestion
  register_taxonomy( 'lion__position', array( 'lion_interview' ), $args );
```
Likewise, this creates a new taxonomy called `taxonomy` rather than `lion__position`.

Reply:
@bryan looks like I didn’t push a commit resolving those issues prior to submitting. Resolved.